Client/ClientRoom.cpp

```diff
@@ -31,10 +32,11 @@
 #include "Transisthor/TransisthorECSLogic/Client/Components/NetworkServer.hpp"
 #include "Transisthor/TransisthorECSLogic/Client/Systems/SendNewlyCreatedToServer.hpp"
 #include "Transisthor/TransisthorECSLogic/Client/Systems/SendToServer.hpp"
-#include "R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateAlliedProjectile.hpp"
+#include "R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateButton.hpp"
```

Comment:
Why replacing AlliedProjectile with Button ?

Reply:
Because Allied was useless, and the CLang format decided to put it in the same line